### DISCUSSION TITLE
[14.0.1] Generating same build result from same build ID is not fully fixed

### DISCUSSION THREAD

**Discussion by prunusnira** [vercel/next.js#58239] [Help]
### Summary

nextjs 14.0.1 + node 18.18.2 here

https://github.com/vercel/next.js/pull/55358 seems fixed the problem, but 'next build' still creates several different file name for same build id.

BUILD_ID file created in .next folder has same content (same build id) but static file names are not same.

(image below: Two servers, which shares same build id has different file name for several files after build)

![image](https://github.com/vercel/next.js/assets/10396020/9acaca58-5d86-4bb7-acca-80f4a79ab9c3)

**Comment by JuanLuisGarciaBorrego** [comment]
Any news ?
